src/snakemake/ioutils/input.py

```
@@ -28,3 +28,20 @@ def extract_checksum(infile, **kwargs):
         )
     except ImportError:
         raise WorkflowError("Pandas is required to extract checksum from file.")
+
+
+def flatten(list_of_lists: List) -> List:
```

Ruff (0.8.2)

Undefined name `List`

(F821)